api/team.go

```diff
@@ -456,6 +456,15 @@ func inviteMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		user = result.Data.(*model.User)
 	}
 
+	var invNum int64 = 0
+	for i, invite := range invites.Invites {
+		if result := <-Srv.Store.User().GetByEmail(c.Session.TeamId, invite["email"]); result.Err == nil || result.Err.Message != "We couldn't find the existing account" {
```

Comment:
Maybe it's better to use result.Err.Where instead if result.Err.Message?
Message text could be changed in feature.